frontend/public/components/utils/_copy-to-clipboard.scss

```diff
@@ -0,0 +1,14 @@
+.copy-to-clipboard {
```

Comment:
We should be following BEM naming. Console uses a `.co-` prefix.

http://getbem.com/naming/